cmd/generate/config/rules/generic.go

```diff
@@ -19,7 +19,7 @@ func GenericCredential() *config.Rule {
 			"password",
 			"auth",
 			"access",
-		}, `[0-9a-z\-_.=]{10,150}`),
+		}, `[0-9a-z\-_.=!#$%&()*+,/:;<>?@[\\\]^{|}~]{10,150}`),
```

Comment:
You need to escape the slash like:
```[0-9a-z\-_.=!#$%&()*+,\/:;<>?@[\\\]^{|}~]{10,150}```

**Original:**
[0-9a-z\-_.=!#$%&()*+,   **===>   /   <===**   :;<>?@[\\\]^{|}~]{10,150}

Reply:
At least according to regex101, the syntax should be fine, also to the unit tests went through. However, I changed it anyway, to be more on the save side.